qiskit_machine_learning/algorithms/classifiers/vqc.py

```diff
@@ -49,8 +50,8 @@ def __init__(
             loss: A target loss function to be used in training. Default is cross entropy.
             optimizer: An instance of an optimizer to be used in training.
             warm_start: Use weights from previous fit to start next fit.
+            input_gradients: Determines whether to compute gradients with respect to input data.
```

Comment:
This can be just set to `False` for VQC. Input gradients make only sense if the QNN is embedded in a larger workflow, VQC has no way to use it (otherwise one could use the NNClassifier and construct it manually).